<launch>
	<arg name="namespace" default="" />

	<!-- Launch femto_bolt.launch from vibro_tactile_toolbox package -->
	<include
		file="$(find vibro_tactile_toolbox)/launch/femto_bolt.launch">
		<arg name="camera_name" value="$(arg namespace)/side_camera"/>
	</include>

	<group ns="$(arg namespace)">
		<!-- Start compressed_image_republisher node -->
		<node name="compressed_image_cropped_republisher" pkg="vibro_tactile_toolbox" type="compressed_image_cropped_republisher.py" output="screen">
			<param name="compressed_image_sub_topic_name" value="side_camera/color/image_raw/compressed" />
			<param name="compressed_image_pub_topic_name" value="side_camera/color/image_cropped/compressed" />
			<param name="x_offset" value="600" />
			<param name="y_offset" value="2150" />
			<param name="x_size" value="1280" />
			<param name="y_size" value="720" />
			<param name="rotation_direction" value="counter_clockwise" />
		</node>

	</group>
</launch>
